### ISSUE TITLE
Android NavigationBar Gone when Tabbed Page set to Material

### ISSUE THREAD

**Issue by ianvink** [xamarin/Xamarin.Forms#5723]
XF 3.6, Android only, Set TabbedPage to Visual="Material", the NavigationBar disappears for children in the TabbedPage when the page in the TabbedPage is a Navigation page.

### Steps to Reproduce
RootPage set to this tabbedpage configuration. Add Navigation pages as children tabs in code behind:

`    <?xml version="1.0" encoding="utf-8"?>

    <TabbedPage
	xmlns="http://xamarin.com/schemas/2014/forms"
	xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
	xmlns:android="clr-namespace:Xamarin.Forms.PlatformConfiguration.AndroidSpecific;assembly=Xamarin.Forms.Core"
	xmlns:ios="clr-namespace:Xamarin.Forms.PlatformConfiguration.iOSSpecific;assembly=Xamarin.Forms.Core"
	android:TabbedPage.ToolbarPlacement="Bottom"
	android:TabbedPage.IsSwipePagingEnabled="False"
	ios:Page.UseSafeArea="True"
       Visual="Material"
	Title="Hi Mom"
	x:Class="Max.Views.Navigation.RootTabbedPage">
      </TabbedPage>
`

Remove the Visual="Material" and it works fine

### Expected Behavior
NavigationBar is visible

### Actual Behavior
NavigationBar is not visible

### Basic Information
 
- Platform Target Frameworks: 
 
  - Android: Newest, 9 . XF 3.6 newest
 

### Screenshots

![2019-03-27 13_32_43-Window](https://user-images.githubusercontent.com/112916/55110248-e17da300-5094-11e9-8bda-f788772d7ef2.png)

![2019-03-27 13_34_29-Window](https://user-images.githubusercontent.com/112916/55110355-1a1d7c80-5095-11e9-9d72-21a3ccffe01a.png)

**Comment by PureWeen** [comment]
@ianvik this will be fixed with the next SR of 3.6
There was a bug that was causing the non appcompat renderers to be used when things like TabbedPage were set to Material

Here's the test I'm running locally to make sure your use case above works

```C#
		public MainPage()
		{
			InitializeComponent();

			var page = new NavigationPage(new ContentPage() { Title = "first page" }) { Title = "navigation page" };
			page.PushAsync(new ContentPage() { Title = "second page", Content = CreateStackLayout(new[] { new Entry() }) });

			this.Children.Add(page);
			this.Children.Add(new NavigationPage(new ContentPage()));
			this.Children.Add(new NavigationPage(new ContentPage()));
		}
```

![image](https://user-images.githubusercontent.com/5375137/55117341-219e4f00-50b0-11e9-9cb8-81c53b65423d.png)

**Comment by samhouts** [comment]
closed by #5676
